.gitignore

```diff
@@ -97,9 +98,11 @@ total.coverage/
 coverage_percent.txt
 
 #build tests
+qa/
```

Comment:
I haven't seen any instances of it being created in new trees, so most probably if you have this, it's a remainder from an old build.